### PR TITLE
added 2 podcasts and added name to contributors

### PR DESCRIPTION
**Pull request by danithang**

### CODE REVIEW

File: Podcasts.md [hunk 1]
```diff
@@ -38,6 +38,7 @@
...
 - [Ladybug Podcast](https://www.ladybug.dev/): This podcast co-hosted by 3 ladies who wanted to add their voices to the space and share their experiences and advice. They have great discussions around how to start coding, the hot technologies right now, how to get your first developer job, and more!
 - [The Stack Overflow Podcast](https://stackoverflow.blog/podcast/): The Stack Overflow podcast is a weekly conversation about working in software development, learning to code, and the art and culture of computer programming. Hosted by Sara Chipps, Paul Ford, and Ben Popper, the series will feature questions from our community, interviews with fascinating guests, and hot takes on what’s happening in tech.
+- [HTML All The Things](https://www.htmlallthethings.com/podcast): Join hosts as they dive head first into the world of web development and design. In this show they discuss running a small business, making websites and web apps for customers from all over the industry.  They take a deep dive on things like their latest projects, their business strategies and much more!
```
[the remark this answers is not in the document]
Comment (danithang) [reply Podcasts.md:41]: Thank you, that is a better description.

File: Podcasts.md [hunk 2]
```diff
@@ -38,6 +38,7 @@
...
 - [Ladybug Podcast](https://www.ladybug.dev/): This podcast co-hosted by 3 ladies who wanted to add their voices to the space and share their experiences and advice. They have great discussions around how to start coding, the hot technologies right now, how to get your first developer job, and more!
 - [The Stack Overflow Podcast](https://stackoverflow.blog/podcast/): The Stack Overflow podcast is a weekly conversation about working in software development, learning to code, and the art and culture of computer programming. Hosted by Sara Chipps, Paul Ford, and Ben Popper, the series will feature questions from our community, interviews with fascinating guests, and hot takes on what’s happening in tech.
+- [HTML All The Things](https://www.htmlallthethings.com/podcast): Join hosts Matt Lawrence and Mike Karan as they dive head first into the world of web development and design. In this show,they discuss running a small business, making websites and web apps for customers from all over the industry.  Matt and Mike also take a deep dive on things like their latest projects, their business strategies, and much more!
```
Comment (CBID2) [inline Podcasts.md:41]:
```suggestion
- [HTML All The Things](https://www.htmlallthethings.com/podcast): Join hosts Matt Lawrence and Mike Karan as they dive into the world of web development and design. In this show, they discuss running a small business and making websites and web apps for customers from all over the industry.  Matt and Mike also take a deep dive into things like their latest projects, their business strategies, and much more! 
```

**Review by CBID2 — APPROVED** [review]
Hi @danithang! :) I just made a few more changes.

File: Podcasts.md [hunk 3]
```diff
@@ -38,6 +38,7 @@
...
 - [Ladybug Podcast](https://www.ladybug.dev/): This podcast co-hosted by 3 ladies who wanted to add their voices to the space and share their experiences and advice. They have great discussions around how to start coding, the hot technologies right now, how to get your first developer job, and more!
 - [The Stack Overflow Podcast](https://stackoverflow.blog/podcast/): The Stack Overflow podcast is a weekly conversation about working in software development, learning to code, and the art and culture of computer programming. Hosted by Sara Chipps, Paul Ford, and Ben Popper, the series will feature questions from our community, interviews with fascinating guests, and hot takes on what’s happening in tech.
+- [HTML All The Things](https://www.htmlallthethings.com/podcast): Join hosts Matt Lawrence and Mike Karan as they dive into the world of web development and design. In this show, they discuss running a small business and making websites and web apps for customers from all over the industry.  Matt and Mike also take a deep dive into things like their latest projects, their business strategies, and much more!
```
Comment (LaurelineP) [inline Podcasts.md:41]: Regarding the following:  
`they discuss running a small business and making websites and web apps for customers from all over the industry.`

Could be improved with the following - avoiding repetitive use of  "and"
`they discuss running a small business, making websites and web apps for customers from all over the industry.`

**Review by LaurelineP — CHANGES_REQUESTED** [review]

File: Podcasts.md (hunk 3 above)
Comment (danithang) [reply Podcasts.md:41]: Yeah that sounds better.

**Review by danithang — COMMENTED** [review]
Changed a sentence to sound better

**Pull request merged**